src/CoreGraphics/CGColorSpace.cs

```diff
@@ -85,6 +86,24 @@ public CGColorSpace (CFPropertyList propertyList)
 			this.handle = CGColorSpaceCreateWithPropertyList (propertyList.Handle);
 		}
 
+#if !NET
+		[Mac (12,0), iOS (15,0), TV (15,0), Watch (8,0), MacCatalyst (15,0)]
+#else
+		[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos12.0"), SupportedOSPlatform ("maccatalyst15.0")]
+#endif
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern IntPtr CGColorSpaceCreateWithColorSyncProfile (/*[NullAllowed] ColorSyncProfileRef */ IntPtr profile, /*[NullAllowed] CFDictionaryRef */ IntPtr options);
+
+#if !NET
+		[Mac (12,0), iOS (15,0), TV (15,0), Watch (8,0), MacCatalyst (15,0)]
+#else
+		[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos12.0"), SupportedOSPlatform ("maccatalyst15.0")]
+#endif
+		public CGColorSpace (IntPtr? profile, NSDictionary options)
```

Comment:
not sure I like a nullable `IntPtr`
ideally we should not expose `IntPtr` and, if we must, it should be straightforward and similar to the low-level API we have

Reply:
Not clear why this is _available_ for iOS since it's only exposed in macOS' `NSColorSpace` and the framework, ColorSync, is not available outside macOS
https://developer.apple.com/documentation/colorsync/1462094-colorsyncprofilecreate?language=objc

I suggest disabling the API, at least for iOS/tvOS/watchOS, until we know It can be somehow used on those platforms.

Reply:
I was in the same feeling, we do not suppose that so might not give too much to users.

Reply:
not sure I parse the last part ?

Reply:
tldr; I agree :)